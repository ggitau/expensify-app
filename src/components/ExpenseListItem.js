import React from 'react';
import {removeExpense} from '../actions/expenses';
import {Link} from 'react-router-dom';
const ExpenseListItem = ({id,description,amount,createdAt}) => (
    <div>
    <h3><Link to={`/edit/${id}`}>{description}</Link></h3>
    <p>{amount}</p>
    <p>{createdAt}</p>

    </div>
);
export default ExpenseListItem;
