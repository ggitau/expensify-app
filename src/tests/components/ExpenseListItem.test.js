import {shallow} from 'enzyme';
import React from 'react';
import ExpenseListItem from '../../components/ExpenseListItem';
import expenses from '../fixtures/expenses';

test('it should render expense list item', () => {
  const wrapper = shallow(<ExpenseListItem {...expenses[0]}/>);
  expect(wrapper).toMatchSnapshot();
});
