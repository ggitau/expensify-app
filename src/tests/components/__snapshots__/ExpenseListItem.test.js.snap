// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render expense list item 1`] = `
<div>
  <h3>
    <Link
      replace={false}
      to="/edit/one"
    >
      gum
    </Link>
  </h3>
  <p>
    195
  </p>
  <p>
    0
  </p>
</div>
`;
